spec/graphql/authorization_spec.rb

```diff
@@ -372,6 +372,9 @@ class Schema < GraphQL::Schema
       if TESTING_INTERPRETER
         use GraphQL::Execution::Interpreter
       end
+      if TESTING_AST_ANALYSIS
+        use GraphQL::Analysis::AST
+      end
```

Comment:
Should this be removed, since we're bailing on that analyzer? (If it should be removed, then I think we can remove TESTING_AST_ANALYSIS altogether, right?)